packages/pds/src/api/com/atproto/identity/updateHandle.ts

```diff
@@ -42,7 +42,9 @@ export default function (server: Server, ctx: AppContext) {
       })
 
       // Pessimistic check to handle spam: also enforced by updateHandle() and the db.
-      const account = await ctx.accountManager.getAccount(handle)
+      const account = await ctx.accountManager.getAccount(handle, {
+        includeDeactivated: true,
+      })
```

Comment:
So we reserve handles of deactivated users but don't resolve them from `resolveHandle`—sound right?